import_export/resources.py

```diff
@@ -1196,9 +1197,17 @@ def field_from_django_field(cls, field_name, django_field, readonly):
 
         FieldWidget = cls.widget_from_django_field(django_field)
         widget_kwargs = cls.widget_kwargs_for_field(field_name, django_field)
+
+        attribute = field_name
+        column_name = field_name
+        # To solve #974
+        if isinstance(django_field, ForeignKey) and "__" not in column_name:
+            attribute += "_id"
+            widget_kwargs["key_is_id"] = True
```

Comment:
Correct me if I am wrong, please, but to me, it feels like this belongs more in `widget_kwargs_for_field` than here.